source/nodejs/adaptivecards-designer/src/card-designer-surface.ts

```diff
@@ -1015,45 +965,21 @@ export class CardDesignerSurface {
         }
     }
 
-    reassignCardElementToCarousel(carouselPeer: DesignerPeers.CarouselPeer) {
-        const carouselPage = carouselPeer.getChildAt(0);
-        if (carouselPage instanceof DesignerPeers.CardElementPeer) {
-            // Filter the root peer's children for all non-carousel peers that are not forbidden child elements
-            // Then, reassign (via tryAdd) each element from the root peer to the current carousel page
-            // Reassigning all elements to the carousel page ensures carousel's singleton behavior
-            const forbiddenChildElements = (carouselPage.cardElement as Adaptive.CarouselPage).getForbiddenChildElements();
-            this.searchAndRemoveForbiddenElements(this._rootPeer, forbiddenChildElements);
-            this._rootPeer.children.filter((child) => 
-                (!(child instanceof DesignerPeers.CarouselPeer)))
-                .forEach((e) => {carouselPage.tryAdd(e);});
-        }
-    }
-
     private searchAndRemoveForbiddenElements(peerToSearch: DesignerPeers.DesignerPeer, forbiddenElements: any) {
```

Comment:
`searchAndRemoveForbiddenElements` is only called by `reassignCardElementsToCarousel`, so this method can be removed as well.